grakn-test/src/test/java/ai/grakn/test/graql/analytics/AnalyticsTest.java

```diff
@@ -127,4 +140,52 @@ public void testNullResourceDoesntBreakAnalytics() throws GraknValidationExcepti
             fail();
         }
     }
+
+    @Test
+    public void testConcurrentJobs() {
```

Comment:
Can we get a more descriptive name? Imagine I broke this test and I need to fix it without bothering you? 
Most of us are following the paradigm of describing the main action and the expected result. For example `whenCallingGetConceptWithAnExistingConceptId_ItReturnsThatConcept`